client/src/components/layout/EditTableForm.js

```diff
@@ -0,0 +1,228 @@
+import React, { useState, useEffect } from "react"
+import { Redirect, Link } from "react-router-dom"
+
+import validateInput from "../../services/validateInput.js"
+import fetchPlayerAndStats from "../../services/fetchPlayerAndStats.js"
+import translateServerErrors from "../../services/translateServerErrors.js"
+import nestStatsUnderPlayer from "../../services/nestStatsUnderPlayer.js"
+import ErrorList from "./ErrorList.js"
```

Comment:
I would make a separation between importing libraries, services, and components through line breaks